### ISSUE TITLE
Botania quest (mana pool and mana spreader)

### ISSUE THREAD

**Issue by sdgdsbjhjfh** [AllTheMods/Spellbound#104]
### What would you like to know?

The quest of mana pool and mana spreader can not be completed even if I had both of them in my inventory, it seems like they can not be detected by the quest even if I create a new world and try it there. Mana pearl/diamond and manasteel ingot can be detected successsfully. I have tried crafting them, throwing them on the ground and pick them up again, and getting them with creative mode, none of them successfully completed the quest.

**Comment by MadWomen** [comment]
you can complete those quests by crafting them in a vanilla crafting bench, there are a few bugs with the tinkers crafting table.

**Comment by ArtOfCombat** [comment]
Can confirm - this does NOT work. I crafted mine in a vanilla crafting table, no dice.

**Comment by Diseaseofdeath** [comment]
Unfortunately the only fix I've found for this is manually setting the quest's progress to completed using /ftbquests change_progress @p complete <quest_object> . For me, the quest's id was 1F06D5DA505ECDE0, but I don't know if this is the same for everyone. I found it by pinning the quest, going into my world's files, and opening the ftbquests SNBT file with Notepad++, and scrolling to the bottom where it shows your pinned quests. Hopefully this helps, I wasn't able to find a way to finish it without turning cheats on via LAN.
